usd/src/UsdTestUtils.hh

```diff
@@ -95,6 +104,57 @@ void CheckPrimPose(const pxr::UsdPrim &_usdPrim,
   EXPECT_TRUE(checkedOpOrder);
 }
 
+/// \brief Compare the Inertial of a USD prim to the desired values
+/// \param[in] _usdPrim The USD prim
+/// \param[in] _targetMass Mass of the link that _usdPrim should have
+/// \param[in] _targetDiagonalInertia Diagonal Inertia that _usdPrim should have
+/// \param[in] _targetCenterOfMass Center of mass that _usdPrim should have
+/// \param[in] _isRigid True if _usdPrim should be a rigid body, False otherwise
+void CheckInertial(const pxr::UsdPrim &_usdPrim,
+    float _targetMass,
+    const pxr::GfVec3f &_targetDiagonalInertia,
+    const pxr::GfVec3f &_targetCenterOfMass,
```

Comment:
the `physics:principalAxes` attribute is relevant to inertials as wells and should be checked in this method. If you aren't going to add support for checking that variable now, please add a `TODO` comment to remind to do it later

Reply:
I added test coverage for principal axes in https://github.com/ignitionrobotics/sdformat/pull/828/commits/f0ff63e1f41a4a590c6124a0e77e3038c04432e3. If this looks good to you, you can mark this as resolved.